### PR TITLE
refactor(website): improvements for bajour 2.0

### PR DESCRIPTION
**Pull request by Itrulia**
Already present in https://next.bajour.ch/

### CODE REVIEW

**Comment by elias-summermatter** [comment]
@Itrulia what is the state here, can we merge this?

File: libs/block-content/website/src/lib/teaser/teaser-grid-block.tsx [hunk 1]
```diff
@@ -48,6 +48,8 @@ export const TeaserGridBlock = ({numColumns, teasers, className}: BuilderTeaserG
         <Teaser
           key={index}
           teaser={teaser}
+          index={index}
```
Comment (Itrulia) [inline libs/block-content/website/src/lib/teaser/teaser-grid-block.tsx:51]: Should maybe done inside `TeaserGridBlock` itself to select the correct `Teaser` but fine IMO for now.

**Review by antkiewiczk — APPROVED** [review]
changes look good, is it ready to merge?

**Comment by Itrulia** [comment]
@antkiewiczk some could be reverted soon with my refactoring at Bajour, so waiting for that.

**Pull request merged**